about.html

```diff
@@ -13,52 +13,98 @@
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+      <header>
+        <img class="logo" src="img/lambda-black.png">
+        <nav>
+          <ul class="nav-links">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Products</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <section class="img-content">
+        <div class"img-container">
+          <img src="img/jumbo-about.png">
+        </div>
+      </section>
+
+      <section class="about-info">
+        <div class="about-container">
+          <h2>About</h2>
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+        </div>
+      </section>
+
+      <section class="image-content">
+
+        <div class="grid-container-images">
+          <div class="grid-tem">
+            <img src="img/about-plan.png" alt="strategy">
+            <div class="grid-item-content">
+              <h2>Strategy</h2>
+              <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            </div>
+            <div class="learn-more">
+              <div>
+                Learn More
+              </div>
+            </div>
+          </div>
+
+          <div class="grid-item">
+            <img src="img/about-working.png" alt="How We Work">
+            <div class="grid-item-content">
+              <h2>How We Work</h2>
+              <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            </div>
+            <div class="learn-more">
+              <div>
+                Learn More
+              </div>
+            </div>
+          </div>
+
+          <div class="grid-item">
+            <img src="img/about-office.png" alt="Places We Work">
```

Comment:
Great job! I love how you remembered to accurately name the alt attribute text for each section img.